core-header-panel.html

```diff
@@ -150,7 +150,7 @@
           <content id="mainContent" select="*"></content>
         </div>
 
-        <div id="dropShadow" class-name="[[_dropShadowClass]]"></div>
+        <div id="dropShadow" hidden$="[[_dropShadowVisibility]]"></div>
```

Comment:
That feature was mentioned in an email last week, and it's now covered in the primer as ["Annotated computed properties"](https://github.com/Polymer/polymer/blob/0.8-preview/PRIMER.md#annotated-computed).